lib/common-lib.sh

```diff
@@ -5,28 +5,27 @@
 
 IFS=$'\n'
 
+# If the repo contains a .formatting-directory file, only files in the specified directories will be returned (one directory per line).
+# If .formatting-directory doesn't exist then all files in the repository are checked.
+function directories_to_check() {
+    locations_to_diff=''
+	[ -e ".formatting-directory" ] && locations_to_diff=$( cat .formatting-directory | sed 's/ /\\ /g' );
+}
+
 # Returns a list of Objective-C files to format.
 # No parameters: return staged ObjC files only
 # Optional parameter: a git SHA. Returns a list of all ObjC files which have changed since that SHA
-# 
-# If the repo contains a .formatting-directory file, only files in the specified directory will be returned.
 function objc_files_to_format() {
 	optional_base_sha="$1"
-
-	location_to_diff=''
-	[ -e ".formatting-directory" ] && location_to_diff=$( cat .formatting-directory );
-
+	directories_to_check
 	# optional_base_sha is intentionally unescaped so that it will not appear as empty quotes.
-	files=$(git diff --cached --name-only $optional_base_sha --diff-filter=ACM -- "$location_to_diff" | grep -e '\.m$' -e '\.mm$' -e '\.h$' -e '\.hh$')
+	files=$(git diff --cached --name-only $optional_base_sha --diff-filter=ACM -- $locations_to_diff | grep -e '\.m$' -e '\.mm$' -e '\.h$' -e '\.hh$')
 	echo "$files" | grep -v 'Pods/' | grep -v 'Carthage/' >&1
 }
 
 # Returns a list of all Objective-C files in the git repository.
```

Comment:
same here, `all` should be qualified to acknowledge the extra configuration which may be in place